app/views/sephora_style_guide/home/_forms.slim

```diff
@@ -35,6 +35,43 @@ article#forms
     .form-group.floating-labels
       label.control-label for='email5' Email (Disabled Field)
       input#email5.form-control type='email' value='testemail' disabled=true
+    
+    br
+    h4.h4 pug
+    pre
+      code.language-pug
+        |
+          .form-group.floating-labels
+            label.control-label(for='email') Email (Empty Field)
+            input#email.form-control(type='email')
```

Comment:
the `id` here is redundant as it doesn't serve a purpose in terms of functionality.

Reply:
It allows the field to be focused when the label is clicked